libs/sdk-backend-bear/src/backend/organization/index.ts

```diff
@@ -0,0 +1,15 @@
+// (C) 2021 GoodData Corporation
+
+import { IOrganization, ISecuritySettingsService } from "@gooddata/sdk-backend-spi";
+import { SDK } from "@gooddata/api-client-bear";
+
+import { SecuritySettingsService } from "./securitySettings";
+
+export class BearOrganization implements IOrganization {
+    constructor(private readonly sdk: SDK, public readonly organizationId: string) {}
```

Comment:
need authApiCall guard here instead of SDK instance (see above)

Reply:
Fixed.